.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: 'master'
+    hooks:
+      - id: autopep8
+        args:
+          - --recursive
+          - --in-place
+          - --aggressive
```

Comment:
Let's use the double `--aggressive` here. I've reviewed the changes made by level 2 and they seem reasonable.

Reply:
aha I thought it is a typo :D

Reply:
added